src/main/java/menion/android/whereyougo/maps/mapsforge/mapgenerator/MapGeneratorFactory.java

```diff
@@ -83,7 +83,7 @@ public static MapGenerator createMapGenerator(MapGeneratorInternal mapGeneratorI
             case ESRI_WORLD_IMAGERY:
                 return new FormatURLTileDownloader(
                         18,
-                        "https://services.arcgisonline.com/arcgis/rest/services/World_Imagery/MapServer/tile/{z}/{y}/{x}.png",
+                        "https://services.arcgisonline.com/ArcGIS/rest/services/World_Imagery/MapServer/tile/{z}/{y}/{x}.png",
```

Comment:
The original address is fine. This is the form wich are used on the server, see at https://services.arcgisonline.com/arcgis/rest/services/World_Imagery/MapServer
The paths are context insensitive.
Please revert.